### PR TITLE
align environment variable handling closer with docker-compose  

### LINKED ISSUE #35: Support integer values of a map service environment

### LINKED ISSUE #39: environment values inconsistency
Environment variables are not persisting their original values, instead, kube-compose is taking on of the environmental values in the docker-compose and reusing it in all instances.

### LINKED ISSUE #42: handle null values in YAML

### PR DESCRIPTION
**Pull request by jbrekelmans**
fixes #39
fixes #42 
fixes #35

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -227,17 +228,27 @@ func parseServiceYAML2_1(serviceYAML *service2_1) (*Service, error) {
 	service.Environment = make(map[string]string, len(serviceYAML.Environment.Values))
 	for _, pair := range serviceYAML.Environment.Values {
 		var value string
+		if len(pair.Name) == 0 {
```
Comment (xUnholy) [inline pkg/config/config.go]: Can probably just check pair.Name == ""

File: pkg/config/schema.go [hunk 2]
```diff
@@ -119,39 +119,72 @@ func (t *dependsOn2_1) Decode(into mapdecode.Into) error {
 
 type environmentNameValuePair struct {
 	Name  string
-	Value *string
+	Value *environmentValue
+}
+
+// TODO https://github.com/jbrekelmans/kube-compose/issues/40 check whether handling of large numbers is consistent with docker-compose
+// See https://github.com/docker/compose/blob/master/compose/config/config_schema_v2.1.json#L418
+type environmentValue struct {
```
Comment (xUnholy) [inline pkg/config/schema.go]: Should we also consider taking these as an array of values?

Comment (jbrekelmans) [reply pkg/config/schema.go]: I think what you are talking about already works, you mean this right:
```yaml
environment:
- VAR1=v1
- VAR2=v2
```

File: pkg/config/config.go (hunk 1 above)
Comment (jbrekelmans) [reply pkg/config/config.go]: Yeah, we're doing this in a lot of places and linter doesn't complain. So prefer doing this for consistency.

**Pull request merged**